test/views/declaration/ExporterDetailsViewSpec.scala

```diff
@@ -136,7 +136,7 @@ class ExporterDetailsViewSpec extends UnitViewSpec with CommonMessages with Stub
 
         view must haveGovukGlobalErrorSummary
         view must containErrorElementWithTagAndHref("a", "#details")
-        view.getElementsByClass("govuk-list govuk-error-summary__list").text() must include(messages("supplementary.namedEntityDetails.error"))
+        view.getElementsByClass("govuk-list govuk-error-summary__list").attr("value") mustBe empty
```

Comment:
Did you consider the approach `view must containErrorElementWithMessage(...)` ?

Reply:
I will consider using this approach going forward